awesome/src/objects/tag.rs

```diff
@@ -8,10 +8,12 @@ use rlua::{self, AnyUserData, Integer, Lua, Table, ToLua, UserData,
            UserDataMethods, Value};
 
 use common::{class::{self, Class, ClassBuilder},
-             object::{self, Object, Objectable},
+             object::{self, Object, ObjectBuilder, Objectable},
              property::Property,
              signal};
 
+use objects::client::{ClientState, Client};
```

Comment:
Group these together with the common:: group above.

I haven't been _real_ strict on this, but generally I've been separating the imports like this:

```
<std imports>

<external crate imports>

<project imports>
```